src/workloads/scale/ManyUpdate.yml

```diff
@@ -0,0 +1,83 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-replication"
+Description: |
+  This workload loads a large number of small documents into a single collection, and then does a
+  small multi-update which touches all of them.  It is intended to measure the impact of the load of
+  replication of many oplog entries on the primary.  Number of documents should be significantly
+  greater than the maximum replication batch size of 5K (50K is a good minimum).
+  We will run this against standalone nodes and single-node replica sets as well as 3-node replica
+  sets to determine if any performance changes are due to replication overhead changes (if only
+  3-node replica sets are affected) or some other reason.
+
+Keywords:
+- RunCommand
+- Loader
+- CrudActor
+- updateMany
+- update
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 1000
+
+Actors:
+- Name: Setup
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &DB test
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand: {dropDatabase: 1}
+  - &Nop {Nop: true}
+  - *Nop
+
+- Name: Loader
+  Type: Loader
+  Threads: 10
+  Phases:
+  - *Nop
+  - Repeat: 1
+    Database: *DB
+    CollectionCount: 1
+    Threads: 10
+    MultipleThreadsPerCollection: true
+    DocumentCount: 5e4
+    BatchSize: 1000
+    Document:  # Documents are approximately 100B in size
+      i: 0
+      string0: {^FastRandomString: {length: 100}}
+  - *Nop
+
+- Name: DocumentUpdater
+  Type: CrudActor
+  Threads: 1
+  Database: *DB
+  Phases:
+  - *Nop
+  - *Nop
+  - Duration: 1 minutes
```

Comment:
It looks like the most recent patch build is still running, but an older one shows you got 22 operations completed, which is barely enough to get a 95th percentile. Do you want to run this longer?

Reply:
Ah, good point, I had it short mostly to reduce the development cycle.  5 minutes would give me a 99th percentile (barely) and a decent 95th percentile, do you think that makes sense?

Reply:
That sounds reasonable to me. There's a fair amount of overhead in setting up the system, so a 5 minute test isn't bad. 10 would likely be fine also.